<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BspZipGuiClass</class>
 <widget class="QDialog" name="BspZipGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BspZipGui 1.2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>bspzip.exe path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="bspzip_path"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browse_bspzip">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BSP file path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bspfile_path"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="browse_bsp_file">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Data Folder Path</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="data_folder_path"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="browse_data_folder">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="extract">
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="embed">
       <property name="text">
        <string>Embed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="console">
     <property name="plainText">
      <string>Created using Qt 5.5.1 and Qt Creator 3.5.1
https://sites.google.com/site/softdevdy/software/bspzipgui
</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
